src/assets.rs

```diff
@@ -88,18 +94,12 @@ impl Asset {
         self.verified
     }
 
-    /// Check if the asset is temporary.
-    pub fn temporary(&self) -> bool {
-        self.temporary
-    }
-
-    /// Changes the asset to be verified and temporary without mint information.
+    /// Changes the asset to be verified without mint information.
```

Comment:
Maybe explain why verified assets don't include mint information (this function is meant for exporting, and you don't want to export min information because it's a secret)